content/20.zksync-protocol/20.contracts/10.l1-contracts/20.shared-bridges.md

```diff
@@ -174,7 +174,7 @@ or the section on [ZK chains](/zk-stack/zk-chains)
 ## Components: Chain Specific Contracts
 
 - A chain might implement its own specific consensus mechanism. This needs its own contracts. Only this contract will be
-  able to submit proofs to the State Transition contract.
+  able to submit proofs to the Chain Type contract.
```

Comment:
What was called previously "State Transition" (without the "manager" part) is the ZK Chain contracts

```suggestion
  able to submit proofs to the ZK Chain contract.
```